test/parallel/test-tls-pause.js

```diff
@@ -37,24 +40,23 @@ const bufSize = 1024 * 1024;
 let sent = 0;
 let received = 0;
 
-const server = tls.Server(options, function(socket) {
+const server = tls.Server(options, common.mustCall((socket) => {
   socket.pipe(socket);
-  socket.on('data', function(c) {
+  socket.on('data', (c) => {
     console.error('data', c.length);
   });
-});
+}));
 
-server.listen(0, function() {
+server.listen(0, () => {
```

Comment:
The callback to `listen` could (should?) also be wrapped in `common.mustCall`.

Reply:
changes done